infrastructure/lib/addons/aws-lbc.ts

```diff
@@ -44,6 +44,9 @@ export class AWSLoadBalancerController extends Construct {
             'eks.amazonaws.com/role-arn': sa.role.roleArn
           }
         },
+        clusterSecretsPermissions: {
+          allowAllSecrets: true  
```

Comment:
Can we configuring namespaced role/rolebinding instead of granting access to all secrets?

Reply:
These are being installed using helm charts from this repository directly - https://github.com/aws/eks-charts/tree/master/stable/aws-load-balancer-controller.
So, it already installs role/role-binding from these charts programatically without access to secrets though starting from 2.4.2 version.
Also these secrets can be in any namespace today as per Infra setup.

Reply:
I was reading that helm chart and it is recommended to limit the scope by adding extra role/rolebinding. 
```
# clusterSecretsPermissions lets you configure RBAC permissions for secret resources
# Access to secrets resource is required only if you use the OIDC feature, and instead of
# enabling access to all secrets, we recommend configuring namespaced role/rolebinding.
# This option is for backwards compatibility only, and will potentially be deprecated in future.
clusterSecretsPermissions:
  # allowAllSecrets allows the controller to access all secrets in the cluster.
  # This is to get backwards compatible behavior, but *NOT* recommended for security reasons
  allowAllSecrets: false
```

Reply:
Yeah but in our case the scope of creation of ingress resources can be in any namespace. So, limiting it to one namespace would not work for us. Also, today the values.yaml doesn't provide us the way to do it for a single namespace. It has to be done outside of helm installation even if we choose to do it.